packages/algob/src/internal/deployer.ts

```diff
@@ -132,7 +132,26 @@ class DeployerBasicMode {
    */
   getSSC (nameApproval: string, nameClear: string): SSCInfo | undefined {
     const resultMap = this.cpData.precedingCP[this.networkName]?.ssc ?? new Map();
-    return resultMap.get(nameApproval + "-" + nameClear);
+    const nestedMap = resultMap.get(nameApproval + "-" + nameClear);
+    if (nestedMap) {
+      // return last pushed element in the map(latest timestamp value)
+      return [...nestedMap][nestedMap.size - 1][1];
+    } else {
+      return undefined;
+    }
```

Comment:
Can we do `return getSSCFromKey(nameApproval + "-" + nameClear);` ? If possible then we won't need to repeat code

Reply:
yes